src/hooks/FetchList/Detail/NoteFetchHook.js

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+import {
+  fetchNotes,
+  fetchOneNote,
+  makeNote,
+} from '../../../services/FetchNotes';
+import { useParams } from 'react-router-dom';
+import { useNoteContext } from '../../../context/NoteContext';
+
+export function fetchAllNotes() {
+  const { allNotes, dispatch } = useNoteContext();
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const renderNotes = async () => {
+      const info = await fetchNotes();
+      dispatch({ type: 'LIST', payload: info });
+      // setData(info);
+      setLoading(false);
+    };
+    renderNotes();
+  }, []);
+
+  return [allNotes, loading, dispatch];
+}
```

Comment:
Remember to return an object instead of an array. In NoteListCom, we are calling note and loading from the fetchAllNotes function, however, we are not returning note here.